src/python/AsyncStageOut/TransferWorker.py

```diff
@@ -193,10 +198,13 @@ def files_for_transfer(self):
         ftscp copyjob per source:destination.
         """
         source_dests = self.source_destinations_by_user()
-        jobs = {}
-        jobs_lfn = {}
-        jobs_pfn = {}
-        jobs_report = {}
+        # Generate dictionary of new jobs.
+        # output is for big files, output_sm is for small files
+        output = {"jobs": {},
+                  "jobs_lfn": {},
+                  "jobs_pfn": {},
+                  "jobs_report": {}}
+        output_sm = output
```

Comment:
Mh.. I still was thinking in future to split queues. For example take a look at this FTS job: https://fts3-pilot.cern.ch:8449/fts3/ftsmon/#/job/956ac4d4-bae2-11e5-90bb-02163e00a39b

Which has 6 small files (file size: <200KiB) and also 6 huge files (file size ~800MB).

I was not sure about few FTS things, for example:
ASO gets 5 files and all file size lets say 101MB. This means it will not use reuse conn. But from all these files only one is 100MB and all other are 200KB.
It would increase number of submissions to FTS. Is there any penalty for us/FTS how much FTS jobs we submit?

Next example which I had in mind is. Lets say we have N new jobs to transfer and here is the logic which wanted to add:
```
smallQueueSize = 0
if fileSize >= 100MB:
    addToBigQueue;
elif fileSize < 100MB:
    if (smallQueue[smallQueueSize].totalSize + fileSize) < 100MB:
        addToSmallQueue[smallQueueSize];
    else:
         smallQueueSize += 1
         addToSmallQueue[smallQueueSize]
```
Of course this will mean that we will have more FTS jobs submitted.